### PR TITLE
Remove references to slsa/v2alpha1 in top level docs.

### PR DESCRIPTION
**Pull request by wlynch**
# Changes

This effectively darklaunches it while we're still working on it. Docs on v2alpha1 still exist in pkg/chains/formats/slsa/README.md.

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

``` release-note
NONE
```

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -51,12 +51,11 @@ Supported keys include:
 
 | Key | Description | Supported Values | Default |
 | :--- | :--- | :--- | :--- |
-| `artifacts.taskrun.format` | The format to store `TaskRun` payloads in. | `in-toto`, `slsa/v1`, `slsa/v2alpha1`| `in-toto` |
+| `artifacts.taskrun.format` | The format to store `TaskRun` payloads in. | `in-toto`, `slsa/v1` | `in-toto` |
 | `artifacts.taskrun.storage` | The storage backend to store `TaskRun` signatures in. Multiple backends can be specified with comma-separated list ("tekton,oci"). To disable the `TaskRun` artifact input an empty string ("").  | `tekton`, `oci`, `gcs`, `docdb`, `grafeas` | `tekton` |
 | `artifacts.taskrun.signer` | The signature backend to sign `TaskRun` payloads with. | `x509`, `kms` | `x509` |
 
 > NOTE:, `slsa/v1` is an alias of `in-toto` for backwards compatibality.
-> NOTE:, `slsa/v2alpha1` is WIP and is likely to undergo multiple changes.
```
Comment (chitrangpatel) [inline docs/config.md:59]: we should also remove the comment from line 70.

Comment (wlynch) [reply docs/config.md:59]: Done!

**Review by chitrangpatel — APPROVED** [review]
LGTM

**Comment by chitrangpatel** [comment]
ping @lcarva

**Review by lcarva — APPROVED** [review]
/lgtm

**Pull request merged**